src/jdk.javadoc/share/classes/jdk/javadoc/doclet/Taglet.java

```diff
@@ -96,6 +96,7 @@ public interface Taglet {
      * @return true if this taglet supports block tags
      * @implSpec This implementation returns the inverse
      * result to {@code isInlineTag}.
+     * @since 15
```

Comment:
Might as well insert a blank line above.

Reply:
Done, looking at the existing source code sometimes I find a blank line and sometimes I don't.

Reply:
Understood; my comment was a mild suggestion, not a requirement. I find it easier to read when `@since` is set off with a blank line.